lcls_tools/data_analysis/archiver.py

```diff
@@ -18,82 +20,97 @@
 TIMEOUT = 3
 
 
+@dataclass()
+class ArchiverData:
+    # todo make this not a union
+    # Currently one list if timestamps are shared by all PVs else a dict of
+    # pv -> timstamps
+    timeStamps: Union[List[datetime], Dict[str, List[datetime]]]
+    values: Dict[str, List]
+
+
 class Archiver(object):
     def __init__(self, machine):
```

Comment:
Same here, add a comment or two on inputs/outputs for the class and use case? i.e. what does machine look like?